Dockerfile-recursor

```diff
@@ -78,4 +79,4 @@ EXPOSE 53/tcp
 # webserver port
 EXPOSE 8082/tcp
 
-ENTRYPOINT ["/usr/local/sbin/pdns_recursor-startup"]
\ No newline at end of file
+ENTRYPOINT ["/usr/bin/tini", "--", "/usr/local/sbin/pdns_recursor-startup"]
```

Comment:
Please add the missing newline at eof :-)

Reply:
fixed I think. I might unfix it by accident at some later point though..